### PR TITLE
Support python 3.10

### PR DESCRIPTION
**Pull request by VMRuiz**

### CODE REVIEW

File: docs/release-notes.rst [hunk 1]
```diff
@@ -10,9 +10,9 @@ Release notes
 itemloaders 1.0.4 (2020-11-12)
 ------------------------------
 
--   When adding a :class:`scrapy.item.Item` object as a value into an
+-   When adding a :class:`scrapy.item.scrapy.Item` object as a value into an
```
Comment (kmike) [inline docs/release-notes.rst:13]: hm, this path doesn't look right

Comment (kmike) [reply docs/release-notes.rst:13]: If that's hard to make it work, it's an option to just remove `:class:` here; there won't be a link, but it's not a big deal in release notes.

Comment (kmike) [reply docs/release-notes.rst:13]: I see that the path works, but I'm not sure why; it looks weird :) Do you have an explanation for it @VMRuiz?

Comment (VMRuiz) [reply docs/release-notes.rst:13]: Yes, it looks weird but thats how its anchored in the Scrapy docs: 

https://docs.scrapy.org/en/latest/topics/items.html#scrapy.item.scrapy.Item

Comment (kmike) [reply docs/release-notes.rst:13]: Thanks @VMRuiz for checking it. I suspect after some cleanup these URLs may change again in future, to become more reasonable, but let's solve it separately, if it happens.

**Pull request merged**